README.md

```diff
@@ -36,99 +61,89 @@ make docker-build docker-push IMG=<some-registry>/kogito-serverless-operator:tag
+
+1. Create a namespace for the building phase
 
 ```sh
 kubectl create namespace kogito-builder
 ```
 
-3. Create a secret
+2. Create a secret for the built image pushing
+
 ```sh
 kubectl create secret docker-registry regcred --docker-server=<registry_url> --docker-username=<registry_username> --docker-password=<registry_password> --docker-email=<registry_email> -n kogito-builder
 ```
 
-4. Build and push your image to the location specified by `IMG`:
+or you directly import your local docker config into your kubernetes cluster:
 
 ```sh
-make container-build container-push IMG=<some-registry>/kogito-serverless-operator:tag
+kubectl create secret generic regcred --from-file=.dockerconfigjson=${HOME}/.docker/config.json --type=kubernetes.io/dockerconfigjson -n kogito-builder
 ```
 
-5. Deploy the controller to the cluster with the image specified by `IMG`:
+### Install your workflow
 
-```sh
-make deploy IMG=<some-registry>/kogito-serverless-operator:tag
-```
-
-6. Create a dedicated Namespace for the test:
+1. Create a dedicated namespace for the workflow:
 
 ```sh
 kubectl create namespace greeting-workflow
 ```
 
-7. Install Instances of Custom Resources:
+2. Install the Serverless Workflow custom resource:
 
 ```sh
 kubectl apply -f config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow.yaml -n greeting-workflow
 ```
 
-8. Default configuration
-
-A configmap called kogito-serverless-operator-builder-config will be created under the kogito-serverless-operator-system namespace when the Operator will be installed, and it contains:
-  
-- DEFAULT_BUILDER_RESOURCE = kogito_builder_dockerfile.yaml
-- DEFAULT_WORKFLOW_DEXTENSION = .sw.json
-- DEFAULT_KANIKO_SECRET_DEFAULT = regcred
-- DEFAULT_REGISTRY_REPO = quay.io/kiegroup
-- kogito_builder_dockerfile.yaml = dockerfile content
+## Cleanup your cluster
 
-For the local development the DEFAULT_REGISTRY_REPO must be changed
+You will need to remove the different Custom Resources you created.
 
-### Test It Out
-1. Install the CRDs into the cluster:
+1. Remove created workflow resources
 
 ```sh
-make install
+kubectl delete -f config/samples/sw.kogito_v1alpha08_kogitoserverlessworkflow.yaml -n greeting-workflow
+kubectl delete namespace greeting-workflow
 ```
 
-2. Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
+2. Remove the build namespace
 
 ```sh
-make run
+kubectl delete namespace kogito-builder
 ```
 
-**NOTE:** You can also run this in one step by running: `make install run`
+3. Remove the operator
 
+```sh
+make undeploy
+```
 
-**NOTE:** Run `make help` for more information on all potential `make` targets
-
-More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
-
-# Development and Contributions 
+## Development and Contributions 
 
 Contributing is easy, just take a look at our [contributors](./CONTRIBUTING.md)'guide.
 
```

Comment:
do we need to add the license here?

Reply:
we have the LICENSE file, so I think indeed we could remove that part

Reply:
removed